JEP.md

```diff
@@ -78,11 +78,11 @@ Example event schema
 
 Schema validation is done in the core telemetry framework that routes events from publishers to sinks, so that each event sink does not need to implement schema validation.
 
-**Open Questions/TBD**:
-
-* List of rules for event schemas
-* How do organizations deploy custom schemas?
-* Where are the schemas for public events stored?
+Schemas are used at two places
+* During the emit run-time, for validating events before publishing to the configured event sink.
+  * The schemas are picked up from a side-car package for runtime validation to avoid the network call.
+* During analysis time, for discovering the available schemas and understanding the fields.
+  * Schema names are resolvable URLs for public events for discovery, and private events need not be resolvable but could be just `com.mycompany.myevent1`
```

Comment:
We should probably say they should be valid URIs rather than any strings, regardless of them being resolvable or not